packages/typescriptlang-org/src/components/IntlLink.tsx

```diff
@@ -1,11 +1,7 @@
 import React from "react"
 import { GatsbyLinkProps, Link, graphql } from "gatsby"
-
-export type AllSitePage = {
-  nodes: {
-    path: string
-  }[]
-}
+import { SitePage } from "../__generated__/gatsby-types";
+export type AllSitePage = { readonly nodes: ReadonlyArray<Pick<SitePage, 'path'>> };
```

Comment:
I just changed this to:

```ts
import { AllSitePageFragment } from "../__generated__/gatsby-types";
export type AllSitePage = AllSitePageFragment["allSitePage"];
```

in case you don't know about the type accessor feature in TS

Reply:
oh. It's nice. thanks your kindness :)